src/util.ts

```diff
@@ -82,3 +84,8 @@ export function clearBusyState(...elements: Element[]) {
     element.removeAttribute("aria-busy")
   }
 }
+
+export function reportError(error: FetchResponse | unknown) {
```

Comment:
We might want a better name for this as `reportError` is an existing global: https://developer.mozilla.org/en-US/docs/Web/API/reportError

Reply:
Can go with `processError` instead?